winter-website-contact-form.css

```diff
@@ -0,0 +1,68 @@
+.contactMeSection {
+	display: flex;
+	margin: 1rem;
+	text-align: center;
+	display: block;
+}
+
+h1.contactMeText {
+	color: #5d87cf;
+	margin: 3rem 5rem;
+	position: relative;
+}
+
+h2.contactMeTextDetails {
+	color: #d4bdff;
+	margin: 3rem 5rem;
+	font-size: 3rem;
+	position: relative;
+}
+
+form {
+	margin: 10%;
+	padding: 2%;
+	color: #000000;
+	width: 40%;
+	background-color: #7373bf;
+	text-align: left;
+}
+
+input[type=text], input[type=email], input[type=tel] {
+	color: rgb(102,102,186, 0.6);
```

Comment:
using rgba here and hex everywhere else. might need to change the rest to rgba or find out how to change alpha using hex